Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -231,12 +232,13 @@ void MolDraw2D::drawMolecules(
     drawMols_.back()->setOffsets(col * panelWidth(), row * panelHeight());
     drawMols_.back()->createDrawObjects();
     if (drawMols_.back()->getScale() < drawMols_[minScaleMol]->getScale()) {
-      minScaleMol = i;
```

Comment:
Oops, sorry.
You could perhaps use drawMols_.size() here rather than carrying j around.  It might be a bit more immediately obvious what the intention was.

Reply:
@DavidACosgrove Thanks for your review, I think it is a good idea and I have made the change you suggested.